src/types/auth.ts

```diff
@@ -1,4 +1,4 @@
-import {AvataaarProps} from "components/Avatar";
```

Comment:
Why is it "Avat_aaa_rProps" instead of "AvatarProps" 👀?

Reply:
because the library used under the hood is called [Avataaars](https://www.npmjs.com/package/avataaars), too :)